modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -251,6 +300,7 @@ const Diagram = ({
 					setZoomOutHandler={setZoomOutHandler}
 				/>
 			</ClayIconSpriteContext.Provider>
+			<div id="culo"></div>
```

Comment:
@garofalog why was this marked as resolved and left in and sent to Brian? We should not be committing code with a static `id` like this.